notebooks/integrations/llama-index/README.md

```diff
@@ -0,0 +1,17 @@
+# LlamaIndex notebooks
+
+This folder contains notebooks that demonstrate how to work with the LlamaIndex data framework using Elasticsearch.
+
+<!-- The following notebooks are available:
```

Comment:
is this commented out markdown intended?

Reply:
Yeah this is just placeholder until the actual notebooks are live